release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -1393,19 +1413,19 @@ If you do not want to update your SQLite database catalog image or migrate your
 For more information, see xref:../operators/admin/olm-managing-custom-catalogs.adoc#olm-catalog-sources-and-psa_olm-managing-custom-catalogs[Catalog sources and pod security admission].
 
 [discrete]
-[id="ocp-4-12-operator-sdk-1-25-1"]
-=== Operator SDK 1.25.0
+[id="ocp-4-12-operator-sdk-1-25-x"]
```

Comment:
Is this x supposed to be a 4?

Reply:
Good question! They are going to release patch releases with OCP 4.12 z-stream releases. 1.25 will stay the same for the life of 4.12, but the patch release will change. I thought that this would give a consistent link. I suppose that `1-25-z` would make more sense?

Reply:
Yeah, good point I do think that `z` is more reflective in this case

Reply:
+1 to using `z`.

Reply:
Done.